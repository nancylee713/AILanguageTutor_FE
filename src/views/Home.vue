<template>
  <div class="home">
    <h1>AI Language Learner</h1>
    <h2 class='welcome'>Welcome to AI Language Learner where we are commited to assisting your language learning needs!</h2>
  </div>
</template>

<script>
export default {
  name: 'home',
  components: {
  },
};
</script>

<style>
  h1 {
    font-size: 2em;
    margin: 40px;
  }

  .welcome  {
    font-size: 1.5em;
    width: 80%;
    margin: auto;
  }
</style>
