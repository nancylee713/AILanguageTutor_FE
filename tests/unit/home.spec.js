import { shallowMount } from '@vue/test-utils';
import Home from '@/views/Home.vue';

describe('Home Component', () => {
  let wrapper; 

  beforeEach(() => {
    wrapper = shallowMount(Home);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});