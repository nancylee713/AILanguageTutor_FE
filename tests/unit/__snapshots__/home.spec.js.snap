// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should match snapshot 1`] = `
<div class="home">
  <h1>AI Language Learner</h1>
  <h2 class="welcome">Welcome to AI Language Learner where we are commited to assisting your language learning needs!</h2>
</div>
`;
